pkg/metrics/metrics.go

```diff
@@ -12,9 +12,20 @@ const (
 	metricControllerValue = "garm_operator"
 	metricNamespace       = "garm_operator"
 	garmAPI               = "api_requests"
+	garmInfo              = "info"
 )
 
 var (
+	GarmJwtExpiresAt = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: metricNamespace,
+		Subsystem: garmInfo,
+		Name:      "jwt_expires_at",
+		Help:      "Expiry date of obtained garm-server JWT",
```

Comment:
```suggestion
		Name:      "jwt_expiration_timestamp_seconds",
		Help:      "The date after which the obtained JWT expires. Expressed as a Unix Epoch Time",
```

I would like to reference certmanager as a good example "how to export expiration dates" via metrics.
For that i would prefer to expose the metrics like suggested (and also export the value in seconds)

example metric from a running cert-manager instance:

```
certmanager_certificate_expiration_timestamp_seconds{issuer_group="cert-manager.io",issuer_kind="Issuer",issuer_name="topolvm-webhook-ca",name="topolvm-mutatingwebhook",namespace="topolvm-system"} 1.737031897e+09
```